mops.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "new-base"
-version = "0.1.1"
```

Comment:
was this ever released?

Reply:
Yep it was (added some info to the readme).